crates/astria-sequencer-relayer/tests/blackbox/helpers/test_sequencer_relayer.rs

```diff
@@ -607,6 +608,7 @@ impl TestSequencerRelayer {
             .and_then(|value: serde_json::Value| serde_json::from_value::<ZPage>(value).ok())
             .map_or("unknown".to_string(), |zpage| zpage.status);
 
+            error!("timed out; context: `{context}`, state: `{state}`, healthz: `{healthz}`");
```

Comment:
This helps diagnose the point at which the test fails, since we can get several further log lines after this point before the panic is printed.